resources/config/locales/en.yml

```diff
@@ -13,6 +13,8 @@ en:
           or: or
           replace: " replace it with this one..."
           maximum_file_size: The maximum file size is %{bytes}.
+          resource_title: Title
+          resource_title_help: Extra information about the file
```

Comment:
Can I suggest that `resource_title_help` be "The title visible to users"?

Maybe?